src/lib/conf/conftypes.h

```diff
@@ -110,6 +110,16 @@ typedef struct struct_magic_decl_t {
  * fetch this option should produce a warning.
  **/
 #define CVFLAG_OBSOLETE  (1u<<0)
+/**
+ * Flag to indicate that an option is undumpable. An undumpable option is
+ * never saved to disk, and is prefixed with __.
+ **/
+#define CVFLAG_NODUMP    (1u<<1)
+/**
+ * Flag to indicate that an option is "invisible". An invisible option
+ * is always undumpable, and we don't tell the controller about it.
```

Comment:
Why not keep these options orthogonal, and say "no dump" and "no read"?

Reply:
Agreed. Blocker for this PR, or blocker for calling Tor#29211 done?

Reply:
Should solve before we call #29211 finished.